deployment/terraform/ssh/ssh.go

```diff
@@ -81,6 +81,18 @@ func (sshc *Client) RunCommand(cmd string) error {
 	return sess.Run(cmd)
 }
 
+// StartCommand starts a given command in a new ssh session. Unlike RunCommand
+// this command does not wait command to finish.
```

Comment:
Please also add this: "This is needed for running commands in the background."

Reply:
done 👍